src/cli.rs

```diff
@@ -135,7 +135,14 @@ pub enum HcScaffoldCommand {
         fields: Option<Vec<FieldDefinition>>,
 
         #[structopt(long)]
-        /// Skip ui generation, overriding any widgets specified with the --fields option
+        /// Skips UI generation for this entry, overriding any specified widgets in the --fields option.
+        ///
+        /// **WARNING**: Opting out of UI generation for one entry type but not for others that are linked
```

Comment:
I'm assuming this warning is not shown when using the CLI in interactive mode, is it? I.e. when I run `hc scaffold entry-type` and I'm asked "do you want this entry type to be visible in the UI" I might not ever see that warning...

Reply:
You're right , the warning is now printed out for the user to see, apart from being visible when using the `--help` flag